addon/components/ui-dropdown-array.js

```diff
@@ -12,7 +12,12 @@ export default UiDropdown.extend({
   updateProperty: function(property) {
     return function() {
       if (property === 'selected') {
-        this.execute('set ' + property, this.get(`${property}.${this.get('find_by')}`));
+        let value = this.get(`${property}.${this.get('find_by')}`);
+        if (Ember.isBlank(value)) {
```

Comment:
Since this inherits from UiDropdown maybe this chunk could be abstracted into a function that both of them could utilize since it's exactly the same in both places.